sqlglot/dialects/clickhouse.py

```diff
@@ -463,48 +463,51 @@ def _parse_function(
             optional_parens: bool = True,
             any_token: bool = False,
         ) -> t.Optional[exp.Expression]:
-            func = super()._parse_function(
+            expr = super()._parse_function(
                 functions=functions,
                 anonymous=anonymous,
                 optional_parens=optional_parens,
                 any_token=any_token,
             )
 
+            func = expr.this if isinstance(expr, exp.Window) else expr
+
             if isinstance(func, exp.Anonymous):
                 parts = self.AGG_FUNC_MAPPING.get(func.this)
+                is_combined = parts and parts[1]
                 params = self._parse_func_params(func)
 
+                if is_combined:
+                    exp_class = exp.CombinedParameterizedAgg if params else exp.CombinedAggFunc
+                else:
+                    exp_class = exp.ParameterizedAgg if params else exp.AnonymousAggFunc
+
+                kwargs = {
+                    "exp_class": exp_class,
+                    "this": func.this,
+                    "expressions": func.expressions,
+                }
+
+                if is_combined:
+                    kwargs["parts"] = parts
```

Comment:
Nit: I think I'd rearrange these a bit to avoid the redundant `is_combined` check:

```suggestion
                kwargs = {
                    "this": func.this,
                    "expressions": func.expressions,
                }
                if is_combined:
                    kwargs["parts"] = parts
                    exp_class = exp.CombinedParameterizedAgg if params else exp.CombinedAggFunc
                else:
                    exp_class = exp.ParameterizedAgg if params else exp.AnonymousAggFunc

                kwargs["exp_class"] = exp_class
```